quantum/quantum_keycodes.h

```diff
@@ -458,6 +459,8 @@ enum quantum_keycodes {
     SAFE_RANGE
 };
 
+#define EEPRST EEPROM_RESET
```

Comment:
How about `EERESET`?

Reply:
`EEP_RST`?

Reply:
Yep, that's better than `EEPRST`, though I'm not sure if I prefer `EEP_RST` or `EERESET`. Maybe the latter because `RESET` exists.

Reply:
Just go with one of the two 😃 I vote for `EERESET`.

Reply:
Another argument in favor of `EERESET`: the EEPROM configuration constants/macros are called `EECONFIG_*`.